### DISCUSSION TITLE
Anyone interested in RPN mode?

### DISCUSSION THREAD

**Discussion by firecat53** [clementwzk/OpenCalc#97] [Ideas]
It's tough to find a decent RPN mode calculator on F-Droid. This app would be a great base to work from! Thoughts?

**Comment by OmlineEditor** [comment]
what is RPN mode?

**Comment by firecat53** [comment]
More efficient way of entering calculations. It's stack based and was used on most of the HP scientific calculators.
[RPN](https://www.calculatorlibrary.com/blog/what-does-rpn-mean-on-a-calculator)

**Comment by clementwzk** [comment]
I don't think this feature is really beneficial. Indeed this method is not very used and would complicate the calculator for not much

**Comment by scattenlaeufer** [comment]
It would be really great for OpenCalc to support RPN. That would make it the only RPN supporting calculator app one could used in good conscience, I have found so far.

**Comment by inson1** [comment]
I wouldnt use that. But I love options xD So if someone wants to build it then why not.

**Comment by clementwzk** [comment]
We can implement that, but I will not do it, open a pull request if you want to work on it

**Reply by inson1** [reply]
That was exactly what I said
